iBurn/BRCDataImporter.m

```diff
@@ -72,6 +72,21 @@ - (instancetype) initWithReadWriteConnection:(YapDatabaseConnection*)readWriteCo
     return self;
 }
 
+- (void) resetUpdates {
+    [self.readWriteConnection readWriteWithBlock:^(YapDatabaseReadWriteTransaction * __nonnull transaction) {
+        
+        [transaction setObject: nil
+                        forKey: @"art"
+                  inCollection: [BRCUpdateInfo yapCollection]];
+        [transaction setObject: nil
+                        forKey: @"camps"
+                  inCollection: [BRCUpdateInfo yapCollection]];
+        [transaction setObject: nil
+                        forKey: @"events"
+                  inCollection: [BRCUpdateInfo yapCollection]];
```

Comment:
I forget if it's possible to get the key from some constant somewhere instead of using a string, but this does the trick for now